servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/IoUringTest.java

```diff
@@ -58,22 +60,25 @@ void ioUringIsNotAvailableOnMacOs() {
     }
 
     @Test
-    @EnabledOnOs(value = { LINUX })
+    @EnabledOnOs(LINUX)
     void ioUringIsAvailableOnLinux() throws Exception {
         EventLoopAwareNettyIoExecutor ioUringExecutor = null;
         try {
             IoUringUtils.tryIoUring(true);
-            assertTrue(IoUringUtils.isAvailable());
+            assumeTrue(IoUringUtils.isAvailable(), "io_uring is unavailable on " +
+                    System.getProperty("os.name") + ' ' + System.getProperty("os.version"));
             IOUring.ensureAvailability();
 
             ioUringExecutor = NettyIoExecutors.createIoExecutor(2, "io-uring");
             assertThat(ioUringExecutor.eventLoopGroup(), is(instanceOf(IOUringEventLoopGroup.class)));
 
             try (ServerContext serverContext = HttpServers.forAddress(localAddress(0))
                     .ioExecutor(ioUringExecutor)
+                    .executionStrategy(offloadNone())
```

Comment:
should this test be parameterized? it should work with and without offloading IIUC?

Reply:
it shouldn't matter for this test, but I added a param anyway